app/views/search/index.api.rsb

```diff
@@ -3,13 +3,15 @@ api.array :results, api_meta(:total_count => @result_set.n_hits,
                              :limit => @search_request.limit) do
   @result_set.each do |record|
     api.result do
-      api.id          record.event_id
-      api.title       record.event_highlighted_title
-      api.type        record.event_type
-      api.url         url_for(record.event_url(:only_path => false))
-      api.description record.event_content_snippets.join("\n")
-      api.datetime    record.event_datetime
-      api.rank        record.rank
+      api.id               record.event_id
+      api.title            record.event_highlighted_title
+      api.type             record.event_type
+      api.url              url_for(record.event_url(:only_path => false))
+      api.description      record.event_content_snippets.join("\n")
+      api.datetime         record.event_datetime
```

Comment:
Could you add a comment that this is deprecated but still exists for backward compatibility and using `last_modified_at`  or `registered_at` explicitly is recommended?

Reply:
fix: [61f2096](https://github.com/clear-code/redmine_full_text_search/pull/119/commits/61f20967cf7d53513c6f1314c37eba69cdf6135d) Thanks!